README.md

```diff
@@ -9,54 +9,54 @@
+|   | [`Function.prototype.toString` revision](https://github.com/tc39/Function-prototype-toString-revision)      | Michael Ficarra                    | 3 | [x] [link](tc39/test262#1163) |
+|   | [`global`](https://github.com/tc39/proposal-global)                                                         | Jordan Harband                     | 3 | [x] [link](tc39/test262#765) |
+|   | [Rest/Spread Properties](https://github.com/tc39/proposal-object-rest-spread)                               | Sebastian Markbage                 | 3 | [ ] [link](tc39/test262#865) |
+|   | [Asynchronous Iteration](https://github.com/tc39/proposal-async-iteration)                                  | Domenic Denicola                   | 3 | [ ] [link](tc39/test262#923) |
+|   | [`import()`](https://github.com/tc39/proposal-dynamic-import)                                               | Domenic Denicola                   | 3 | [ ] [link](tc39/test262#1164) |
+|   | [RegExp Lookbehind Assertions](https://github.com/tc39/proposal-regexp-lookbehind)                          | Daniel Ehrenberg                   | 3 | [x] [link](tc39/test262#999) |
+|   | [RegExp Unicode Property Escapes](https://github.com/tc39/proposal-regexp-unicode-property-escapes)         | Brian Terlson, Daniel Ehrenberg, Mathias Bynens | 3 | [x] [link](tc39/test262#1014) |
+|   | [RegExp named capture groups](https://github.com/tc39/proposal-regexp-named-groups)                         | Daniel Ehrenberg, Brian Terlson    | 3 | [x] [link](tc39/test262#998) |
+|   | [`s` (`dotAll`) flag for regular expressions](https://github.com/mathiasbynens/es-regexp-dotall-flag)       | Mathias Bynens, Brian Terlson      | 3 | [x] [link](tc39/test262#997) |
+|   | [Legacy RegExp features in JavaScript](https://github.com/tc39/proposal-regexp-legacy-features)             | Mark Miller, Claude Pache          | 3 | [ ] [link](tc39/test262#1165) |
+|   | [`Promise.prototype.finally`](https://github.com/tc39/proposal-promise-finally)                             | Jordan Harband                     | 3 | [ ] [link](tc39/test262#866) |
+|   | [BigInt](https://github.com/tc39/proposal-bigint)                                                           | Daniel Ehrenberg                   | 3 | [ ] [link](tc39/test262#1056) |
+|   | [Class Fields](https://github.com/tc39/proposal-class-fields)                                               | Daniel Ehrenberg, Jeff Morrison    | 3 | [ ] [link](tc39/test262#1055) |
+|   | [Optional catch binding](https://github.com/tc39/proposal-optional-catch-binding)                           | Michael Ficarra                    | 3 | [ ] [link](tc39/test262#1166) |
+|   | [`function.sent` metaproperty](https://github.com/allenwb/ESideas/blob/master/Generator%20metaproperty.md)  | Allen Wirfs-Brock                  | 2 | [add new](https://github.com/tc39/test262/issues/new) |
```

Comment:
I think the link definitely should go at the top or bottom of the table, and not in every column.